frontend/src/concepts/pipelines/content/configurePipelinesServer/ConfigurePipelinesServerModal.tsx

```diff
@@ -86,31 +75,18 @@ export const ConfigurePipelinesServerModal: React.FC<ConfigurePipelinesServerMod
         createPipelinesCR(namespace, spec)
           .then(() => {
             onBeforeClose();
-            fireFormTrackingEvent(serverConfiguredEvent, {
-              outcome: TrackingOutcome.submit,
-              success: true,
-            });
           })
           .catch((e) => {
             setFetching(false);
             setError(e);
-            fireFormTrackingEvent(serverConfiguredEvent, {
-              outcome: TrackingOutcome.submit,
-              success: false,
-              error: e,
-            });
+
             // Cleanup created password secret
             deleteSecret(project.metadata.name, ExternalDatabaseSecret.NAME);
           });
       })
       .catch((e) => {
         setFetching(false);
         setError(e);
-        fireFormTrackingEvent(serverConfiguredEvent, {
-          outcome: TrackingOutcome.submit,
-          success: false,
-          error: e,
-        });
```

Comment:
Probably a bad rebase, but we definitely want to keep the `fireFormTrackingEvent` calls